### PR TITLE
Testament pre parallel

### PR DESCRIPTION
**Pull request by arnetheduck**
#8978 still needs some love and attention but I don't have as much time for it as I'd like right now - this little PR will help reduce conflicts by doing some of the more trivial changes first:

* moving up one dir
* including the (somewhat unrelated) osproc fix

### CODE REVIEW

File: tests/testament/tester.nim [hunk 1]
```diff
@@ -474,7 +474,7 @@ proc main() =
   case action
```
Comment (Araq) [inline tests/testament/tester.nim]: Why does this file exist and wasn't moved over to ``testament/tester.nim``?

Comment (arnetheduck) [reply tests/testament/tester.nim]: oops, not sure how that happened. fixed.

**Comment by Araq** [comment]
Hmm, why does this have a conflict too complex to resolve for github's UI?

**Comment by arnetheduck** [comment]
no idea. perhaps ui doesn't like file moves? command line merged without issue.

**Pull request merged**